20/homework_20.4.py

```diff
@@ -0,0 +1,11 @@
+import sqlite3
+
+with sqlite3.connect("homework20.sqlite") as connection:
+    cursor = connection.cursor()
+    query = ('CREATE TABLE IF NOT EXISTS users ('
+             'id INTEGER PRIMARY KEY,'
+             'first_name TEXT UNIQUE,'
```

Comment:
`UNIQUE` here doesn't allow to add a first_name that already exists. For example, `John Snow` and `John Walker` are different names, but your table doesn't allow to add another John here.

To check for unique combinations you can use UNIQUE on the table level and set columns that should be unique together:
```
...
first_name TEXT,
last_name TEXT,
age INTEGER,
UNIQUE(first_name, last_name))

Reply:
@Bilyk-VM you changed UNIQUE constraint in _20.4 file but didn't change it here as well